src/release/breaking-changes/describe-enum.md

```diff
@@ -79,7 +79,7 @@ API documentation:
 * [`describeEnum` main][]
 * [`EnumProperty` stable][]
 * [`EnumProperty` main][]
-* [enhanced enums]: https://github.com/dart-lang/language/blob/main/working/0158%20-%20Enhanced%20Enum/proposal.md
+* [enhanced enums]: {{site.github}}/dart-lang/language/blob/main/working/0158%20-%20Enhanced%20Enum/proposal.md
```

Comment:
@parlough 

This doesn't show on the [page](https://docs.flutter.dev/release/breaking-changes/describe-enum#references)

![Screenshot 2023-08-22 at 17 00 59](https://github.com/flutter/website/assets/48603081/1542f0f5-d0d3-4368-9da9-7bbd069562f3)

Reply:
Thanks for pointing that out!

I remember seeing this and I thought I fixed it, but that must have just been in my head or something I planned as follow up :P 

I opened https://github.com/flutter/website/pull/9281 with a fix :)